code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -236,6 +278,111 @@ datum/reagent/medicine/anti_toxin/on_mob_life(var/mob/living/M as mob)
 	..()
 	return
 
+datum/reagent/medicine/defib
+	name = "Exstatic mixture"
+	id = "defib"
+	description = "An amazing chemical that can bring the dead back to life!"
+	color = "#C8A5DC" // rgb: 200, 165, 220
+datum/reagent/medicine/defib/on_mob_life(var/mob/living/M as mob)
+	if(M.stat == DEAD)
+		M.setCloneLoss(0)
+		M.setOxyLoss(0)
+		M.radiation = 0
+		M.heal_organ_damage(5,5)
+		M.setToxLoss(-5)
+		M.hallucination = 0
+		M.setBrainLoss(0)
+		M.disabilities = 0
+		M.sdisabilities = 0
+		M.eye_blurry = 0
+		M.eye_blind = 0
+		M.SetWeakened(0)
+		M.SetStunned(0)
+		M.SetParalysis(0)
+		M.dizziness = 0
+		M.drowsyness = 0
+		M.stuttering = 0
+		M.confused = 0
+		M.sleeping = 0
+		M.jitteriness = 0
+		M.stat = 1
+		..()
+		return
+	else
+		M.adjustBrainLoss(100)//boom
+		M.setToxLoss(100)
+		M.adjustBruteLoss(100)
+		return
+
+datum/reagent/medicine/sodiumf
+	name = "Sodium fluoride"
+	id = "sodiumf"
+	description = "A powerful antitoxin"
+	color = "#C8A5DC" // rgb: 200, 165, 220
+
+datum/reagent/medicine/sodiumf/on_mob_life(var/mob/living/M as mob)
+	if(M.stat != DEAD)
```

Comment:
These DEAD stat checks really aren't needed, it doesn't process if the guy is dead anyways.